AppCenterDistribute/AppCenterDistribute/MSDistributePrivate.h

```diff
@@ -137,13 +142,21 @@ static NSString *const kMSTesterAppUpdateSetupFailedKey = @"MSTesterAppUpdateSet
  */
 @property(nonatomic) id<MSCustomApplicationDelegate> appDelegate;
 
+/**
+ * Authentication session instance.
+ */
 @property(nullable, nonatomic) SFAuthenticationSession *authenticationSession API_AVAILABLE(ios(11.0));
 
 /**
  * Distribute info tracking component which adds extra fields to logs.
  */
 @property(nonatomic) MSDistributeInfoTracker *distributeInfoTracker;
 
+/**
+ * Update track.
```

Comment:
Can we be more elaborate here? 
```suggestion
 * A property to store the in-app updates track mode.
```